ios/FluentUI/PersonaButton/PersonaButtonModifiers.swift

```diff
@@ -6,9 +6,4 @@
 import SwiftUI
 
 public extension PersonaButton {
```

Comment:
If this is more or less empty, do we still need the file?

Reply:
Right now no, but this file needs to be fleshed out in order to support full SwiftUI integration for `PersonaButton`. Same goes for the other empty modifier files: if I remove them now, we'll only add them back later, so I figured I'd leave them where they are for future development.

Reply:
Makes sense to me.